tests/chainerx_tests/unit_tests/routines_tests/test_creation.py

```diff
@@ -1174,3 +1176,57 @@ def test_copy(xp, shape, dtype, device, is_module):
         return xp.copy(a)
     else:
         return a.copy()
+
+
+@chainerx.testing.numpy_chainerx_array_equal()
+@pytest.mark.parametrize('N,M,k', [
+    (2, 1, -2),
+    (2, 1, -1),
+    (2, 1, 0),
+    (2, 1, 1),
+    (2, 1, 2),
+    (3, 4, -4),
+    (3, 4, -1),
+    (3, 4, 1),
+    (3, 4, 4),
+    (6, 3, 1),
+    (6, 3, -1),
+    (3, 6, 1),
```

Comment:
Some seem rather duplicated such as (3, 4, 1) and (3, 6, 1). Could you ensure the coverage while reducing these duplications? Please correct me if I'm wrong.

Reply:
Yes, I think they can be reduced. I took the parametrization from `test_eye`.